week06_policy_based/reinforce_pytorch.ipynb

```diff
@@ -361,7 +336,7 @@
     "\n",
     "    # cast everything into torch tensors\n",
     "    states = torch.tensor(states, dtype=torch.float32)\n",
-    "    actions = torch.tensor(actions, dtype=torch.int32)\n",
+    "    actions = torch.tensor(actions, dtype=torch.long)\n",
```

Comment:
I think `long` is less explicit than `int32` (I don't remember off the top of my head if it's `int32` or `int64`) -- maybe let's keep this explicit?

Reply:
Sure, now it's explicit